react/features/variant/V3C/content.jsx

```diff
@@ -0,0 +1,224 @@
+                    selectMode={selectMode}
+                    setSelectMode={setSelectMode}
+                    currentFill={currentFill}
+                    exome={exome}
+                    density={density}
+                    contentWidth={contentWidth}
+                    manual={manual}
+                    setManual={setManual}
+                    inconsistent={inconsistent}
+                    QCfail_stroke={QCfail_stroke}
+                    QCfail_overlay={QCfail_overlay}
+                    dataExternal={dataExternal}
+                    external_overlay={external_overlay}
+                    raise={raise}
+                    setRaise={setRaise}
+                />
+                <Info 
+                    chr={chr}
+                    pos={pos}
+                    vRef={vRef}
+                    vAlt={vAlt}
+                    dataTotal={dataTotal}
+                    dataVisible={dataVisible}
+                    dataSelected={dataSelected}
+                    setDataTotal={setDataTotal}
+                    setDataVisible={setDataVisible}
+                    setDataSelected={setDataSelected}
+                    setDataLocation={setDataLocation}
+                    color={color}
+                    rename={rename}
+                    fill={fill}
+                    date={date}
+                    manual={manual}
+                    setManual={setManual}
+                    selectMode={selectMode}
+                    currentFill={currentFill}
+                    contentWidth={contentWidth}
+                />
+            </div>
+            <Panel 
+                chr={chr}
+                pos={pos}
+                vRef={vRef}
+                vAlt={vAlt}
+                dataTotal={dataTotal}
+                dataVisible={dataVisible}
+                dataSelected={dataSelected}
+                setDataTotal={setDataTotal}
+                setDataVisible={setDataVisible}
+                setDataSelected={setDataSelected}
+                color={color}
+                rename={rename}
+                selectMode={selectMode}
+                currentFill={currentFill}
+                setCurrentFill={setCurrentFill}
+                manual={manual}
+                exome={exome}
+                setExome={setExome}
+                density={density}
+                setDensity={setDensity}
+                contentWidth={contentWidth}
+                inconsistent={inconsistent}
+                setInconsistent={setInconsistent}
+                QCfail_stroke={QCfail_stroke}
+                setQCfail_stroke={setQCfail_stroke}
+                QCfail_overlay={QCfail_overlay}
+                setQCfail_overlay={setQCfail_overlay}
+                dataExternal={dataExternal}
+                external_overlay={external_overlay}
+                setExternal_overlay={setExternal_overlay}
+                setRaise={setRaise}
+            />
+        </div>
+    )
+}
+
+export default Content;
```

Comment:
Let's automatically add newlines in the ends of files. If you use VSCode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Great!